test/test_config.yaml

```diff
@@ -27,7 +27,7 @@ irods:
   user: wut
   pass: wut
   host: wut.iplantc.org
-  port: 1247
+  port: "1247"
```

Comment:
This was necessary because the current configuration template uses a string. Without the quotes, an error message was generated when we tried to merge the configuration with the defaults.